db/migrate/20230311021131_books.rb

```diff
@@ -0,0 +1,14 @@
+class Books < ActiveRecord::Migration[7.0]
+  def change
+    create_table :books do |t|
+      t.integer :sequence
+      t.string :title
+      t.string :en_title
+      t.string :en_short_summary
+      t.text :en_synopsis
+      t.text :artwork_url
+
+      t.timestamps
```

Comment:
Overall, I am pretty happy with this, but there are some things we should consider. 

1. We shouldn't be able to create a book that doesn't have a `sequence` because then.... we won't have a way to organize the books in order. 

2. We also shouldn't be able to create a Book without title (Gurmukhi title)

Everything else is okay. If a book doesn't have an image, or a summary, it's not the end of the world. 

Therefore, we'd want to do something like `:null => false` on them.

Reply:
I've removed the extra file and resolved the missing null constraints.